pkgs/development/libraries/wlroots/default.nix

```
@@ -18,7 +18,7 @@ stdenv.mkDerivation rec {
   # $out for the library and $examples for the example programs (in examples):
   outputs = [ "out" "examples" ];
 
-  nativeBuildInputs = [ meson ninja pkg-config wayland ];
+  nativeBuildInputs = [ meson ninja pkg-config wayland-scanner ];
```

I thought this change would be required (which would be very problematic) but luckily it seems like the "bin" output is always added, even if `wayland` isn't in `nativeBuildInputs`. In other words the following still works because `wayland` is in `buildInputs` (but it obviously doesn't work for cross-compilation - both with and without this PR):
```suggestion
  nativeBuildInputs = [ meson ninja pkg-config ];
```

So I guess this shouldn't break anything then.

---

This part of the patch only serves as an example btw to demonstrate this.